### ISSUE TITLE
Metadata Fields for File created

### ISSUE THREAD

**Issue by sigmapointindia** [freelancer4you/godigital#3]
Provision to choose and create file creation fields per department.

**Comment by freelancer4you** [comment]
this also means to make the table and filters dynamic this will take I think 3 days or even more, because this is not that easy

**Comment by sigmapointindia** [comment]
Yes this one of those parts which will make the file management system app reliable to use. Instead of giving the flexibility of creating fields, we can give about 50 fields for the client to choose for their department(s). 
The fields can be selected in the beginning during installation phase itself and can be assigned to Departments. During file creation, the modal can be like a landscape as in the screen shot here. Make it less complex as per your ease.
![sap migo screen](https://user-images.githubusercontent.com/8864976/32291066-f7c55e20-bf61-11e7-8e12-30290ed9a41b.png)
